src/aliceVision/multiview/resection/resectionLORansac_test.cpp

```diff
@@ -223,11 +223,12 @@ BOOST_AUTO_TEST_CASE(P3P_Ransac_noisyFromImagePoints)
     ALICEVISION_LOG_DEBUG("Angular error: " << angError);
     ALICEVISION_LOG_DEBUG("Baseline error: " << (Test - Tgt).squaredNorm());
 
+    auto camera = std::make_unique<camera::Pinhole>(0, 0, 1, 1, 0, 0);
     geometry::Pose3 pose = geometry::poseFromRT(Rest, Test);
     refinePoseAsItShouldbe(pts3D,
                            pts2Dnorm,
                            inliers,
-                           new camera::Pinhole(0, 0, 1, 1, 0, 0),
+                           camera.get(),
                            pose,
                            true,
                            false );
```

Comment:
Do we really need the unique_ptr?
```
    camera::Pinhole camera(0, 0, 1, 1, 0, 0);
    geometry::Pose3 pose = geometry::poseFromRT(Rest, Test);
    refinePoseAsItShouldbe(pts3D,
                           pts2Dnorm,
                           inliers,
                           &camera,
                           pose,
                           true,
                           false );
```

Reply:
Indeed, I fixed this issue without thinking :see_no_evil:  This is now fixed.